RAP_Manc.qmd

```diff
@@ -16,60 +16,129 @@ format:
+
+## RAP knowledge-shares
+
+## RAP survey
+
+::: columns
+::: {.column width="60%"}
+
+- Department-wide survey
+- 135 responses
+- All analysts of any profession eligible to respond
+- Responses for each area analysed by their RAP Champion (more on that coming up!)
+- Used to work out how to move forward with implementation
 
 :::
-::: {.column width="30%"}
-[![](dfe-analytical-services.png){fig-align="right"}](https://github.com/dfe-analytical-services)
+
+::: {.column width="10%"}
+
 :::
-::::
 
 
-## [dfeR](https://github.com/dfe-analytical-services/dfeR)
+::: {.column width="30%"}
 
-:::: {.columns}
-::: {.column width=20%}
-[![](dfer-logo.png){height="7.2em"}](https://github.com/dfe-analytical-services/dfeR)
-:::
-::: {.column width=80%}
+[![](rap-survey-1.png){fig-align="right"}]()
 
-### Succinct functions that saves coding effort and make analysts lives easier!
+[![](rap-survey-2.png){fig-align="right"}]()
 
-dfeR should provide simple code to make it easier for analysts to work effectively and consistently.
 :::
-::::
+:::
 
-## Intro to the team
 
-## EES & data screener
+## Champions network
+::: columns
+::: {.column width="50%"}
 
-## Internal standards (good, great and best)
+- We currently have 20 RAP champions across the department
+- Varying professions and based across different groups/areas
+- Not all champions work in areas with statistics production teams
+- Monthly meetings for discussions about progress, presenting results, knowledge sharing and training so we can work towards a shared goal
+- All champions have the opportunity to input into next steps 
 
-## Self-assessment tool
+:::
+::: {.column width="10%"}
+:::
 
-## RAP knowledge-shares
+::: {.column width="40%"}
+[![](rap-champs-1.png){fig-align="right" height="10.2em" width="13.2em"}]()
+[![](rap-champs-2.png){fig-align="right" height="9.2em" width="13.2em"}]()
+:::
+:::
 
-## Champions network 
 
 ## Freestyle RAP sessions
 
+::: columns
+::: {.column width="50%"}
+
+- Started as a result of RAP survey responses and suggestions from champions 
+- Monthly drop-in sessions for people to ask for RAP help
+- Loads of varied questions!
+- Any analyst can join
+- Each month we have a different RAP champion involved to build skills
+
+:::
+::: {.column width="10%"}
+:::
+
+::: {.column width="40%"}
+[![](freestyle-rap.png){fig-align="right"}]()
+:::
+:::
+
 ## Implementation plan
 
 ## Hints and tips
 
+::: columns
+::: {.column width="60%"}
+
+- Build an internal champions network!
+- Have a clear argument for WHY we're doing this
+- Get line managers on board
+- Make communications fun and engaging (emojis work well)
+- Focus on RAP as proportionate to the project
+- If teams are overwhelmed, encourage focus on one element at a time 
+- Keep best practice examples to share
+- Make what you're doing as visible as possible
+
```

Comment:
I think we could add having your own internal standards here? (like our good, great, best), and also just sharing RAP as a 'best practice' thing that everyone can do instead of focusing on one project at a time type thing?